velox/expression/tests/SwitchRewriteTest.cpp

```diff
@@ -25,25 +25,36 @@ class SwitchRewriteTest : public expression::test::SpecialFormRewriteTestBase {
 };
 
 TEST_F(SwitchRewriteTest, basic) {
```

Comment:
lets remove this whole test since its not adding any value till the switch rewrite is added.